File: articles/algorithms-in-public-policy-and-governance/typescript/procedural_readiness.ts
const dueProcess = 0.58;
const transparency = 0.52;
const humanReview = 0.60;
const appealReadiness = 0.54;
const score = (dueProcess + transparency + humanReview + appealReadiness) / 4.0;
console.log(`procedural_readiness_score=${score.toFixed(4)}`);

File: articles/algorithms-in-public-policy-and-governance/c/procedural_readiness.c
#include <stdio.h>

int main(void) {
    double due_process = 0.58;
    double transparency = 0.52;
    double human_review = 0.60;
    double appeal_readiness = 0.54;
    double score = (due_process + transparency + human_review + appeal_readiness) / 4.0;
    printf("procedural_readiness_score=%.4f\n", score);
    return 0;
}
